vagrant/provisioning/roles/minio/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+
+- name: create minio group
+  become: yes
+  group:
+    name: minio
+    state: present
+
+- name: create minio user
+  become: yes
+  user:
+    name: minio
+    group: minio
+    shell: /sbin/nologin
+    home: /home/minio
+
+- name: create minio bin directory
+  become: yes
+  file:
+    path: "{{root_folder}}/app/minio/{{ item }}"
+    state: directory
+    owner: minio
+  with_items:
+    - "bin"
+    - "data"
+
+- name: check if minio file exists
+  become: yes
+  stat: 
+   path: "{{root_folder}}/app/minio/bin/minio"
```

Comment:
It has to follow the standard folder structure.

`{{ root_folder }}/app/minio`, `{{ root_folder }}/data/minio`, `{{ root_folder }}/log/minio`, `{{ root_folder }}/tmp/minio`.

Reply:
Most likely the logging configuration needs to be modified, to ensure the log files go into `{{ root_folder }}/log/minio`.

Reply:
sure, will make changes as per the suggested structure.

Reply:
made changes ads per the suggestion